notebooks/official/pipelines/google_cloud_pipeline_components_bqml_text.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<p>I see you write out the SQL queries to files, but never use the files?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>I was thinking about the best practices. Probably I would have all .sql scripts for each component of  BQ pipeline. In a way that I can load them when I build it</p>